src/BeadingStrategy/CenterDeviationBeadingStrategy.cpp

```diff
@@ -1,69 +1,86 @@
-//Copyright (c) 2021 Ultimaker B.V.
-//CuraEngine is released under the terms of the AGPLv3 or higher.
+// Copyright (c) 2021 Ultimaker B.V.
+// CuraEngine is released under the terms of the AGPLv3 or higher.
+#include <algorithm>
 
 #include "CenterDeviationBeadingStrategy.h"
 
 namespace cura
 {
-    CenterDeviationBeadingStrategy::CenterDeviationBeadingStrategy(const coord_t pref_bead_width, const AngleRadians transitioning_angle, const Ratio wall_split_middle_threshold, const Ratio wall_add_middle_threshold)
-    : BeadingStrategy(pref_bead_width, pref_bead_width / 2, transitioning_angle)
-    , minimum_line_width_split(pref_bead_width * wall_split_middle_threshold)
-    , minimum_line_width_add(pref_bead_width* wall_add_middle_threshold)
-    {
-        name = "CenterDeviationBeadingStrategy";
-    }
+CenterDeviationBeadingStrategy::CenterDeviationBeadingStrategy(const coord_t pref_bead_width,
```

Comment:
If you're doing this, then please put all arguments & brackets on their own line. This isn't python!